src/main/java/bio/terra/service/filedata/flight/ingest/FileIngestBulkFlight.java

```diff
@@ -114,18 +114,13 @@ public FileIngestBulkFlight(FlightMap inputParameters, Object applicationContext
     //    billing information remains valid.
     // 1. Lock the load tag - only one flight operating on a load tag at a time
     // 2. TODO: reserve a bulk load slot to make sure we have the threads to do the flight; abort
-    // otherwise (DR-754)
+    //    otherwise (DR-754)
```

Comment:
According to JIRA this bug is closed. Does the comment still apply?

Reply:
Not sure. Was just reformatting after spotless ~~ruined everything~~ reformatted comments